app/models/user.rb

```diff
@@ -242,7 +242,7 @@ def followed_articles
   def cached_following_users_ids
     RedisRailsCache.fetch(
       "user-#{id}-#{updated_at}-#{following_users_count}/following_users_ids",
```

Comment:
I think we can make this change, but I think we can also fix the "expire frequently" bug because these are caches which inherently don't need to expire so frequently.

Changing `updated_at` to `last_followed_at` (and removing `updated_at` in the podcast key which has both), in addition to the expiration changes, should go a long way.

Reply:
Done! Good call!